### PR TITLE
Feat: Product list

### PR DESCRIPTION
**Pull request by sri-rishi**
Product List contain:

- Navbar
- Filter Section
- Product List Section

### CODE REVIEW

File: components-styles/productList.css [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+.img-box {
+    max-height: 12rem;
+    padding: 1rem;
+} 
+
+.product-card {
+    max-height: 25rem;
+}
+
+@media only screen and (max-width: 1240px) {
```
Comment (manish-GH) [inline components-styles/productList.css:99]: It's recommended to use em for media queries. You could consider changing it for all the media queries

**Review by manish-GH — COMMENTED** [review]
You could consider adding another media query for large screen
Screenshot (2)
s

The UI looks clean, nice work

File: components/productList.html [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+                        <label for="price-l-to-h">
+                            <input class="filter-inputs" type="radio" id="price-l-to-h" name="price-l-to-h" aria-label="Price Low to High">
+                            Price - Low to High
+                        </label>
+                    </li>
+                    <li>
+                        <label for="price-h-to-l">
+                            <input class="filter-inputs" id="price-h-to-l" type="radio" name="price-h-to-l"  aria-label="Price High to Low">
+                            Price - High to Low
+                        </label>
```
Comment (yaswanthmyneni) [inline components/productList.html:92]: display only one among them.
Screenshot 2022-02-17 at 12 22 19 PM

**Review by omkarpatke — APPROVED** [review]
your code looks good to me can't see any suggestion.

File: components/productList.html (hunk 2 above)
Comment (yaswanthmyneni) [reply components/productList.html:92]: I mean both are selected at the same time.

Comment (yaswanthmyneni) [reply components/productList.html:92]: The remaining looks good, you can merge.

**Review by manish-GH — APPROVED** [review]

File: components/productList.html (hunk 2 above)
Comment (sri-rishi) [reply components/productList.html:92]:
```suggestion
                        </label>
```
```suggestion
                        </label>
```

That is functionality  which I will add in reactjs

**Review by Anurag021 — APPROVED** [review]
Code looks good to me, also ui is clean too.

File: components/productList.html (hunk 2 above)
Comment (yaswanthmyneni) [reply components/productList.html:92]: ok

**Review by yaswanthmyneni — APPROVED** [review]
you can merge.

**Pull request merged**